src/js/chartAuthoring.js

```diff
@@ -133,31 +153,77 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             onChartAuthoringInterfaceReady: null,
             onPanelReady: null,
             onPieChartReady: null,
+            onSonifierReady: null,
             onToolReady: {
                 events: {
                     onPanelReady: "onPanelReady",
-                    onPieChartReady: "onPieChartReady"
+                    onPieChartReady: "onPieChartReady",
+                    onSonifierReady: "onSonifierReady"
                 },
                 args: ["{that}"]
-            }
+            },
+            onDataAppliedToDataEntryPanel: null
         },
         listeners: {
-            "onToolReady.addAriaConnections": "floe.chartAuthoring.addAriaConnections"
+            "onToolReady.addAriaConnections": "floe.chartAuthoring.addAriaConnections",
+            "onToolReady.bindPlayClick": {
+                "this": "{that}.dom.sonifierPlay",
+                "method": "click",
+                "args": ["{chartAuthoringInterface}.sonifier.events.onSonificationRequested.fire"]
+            },
+            "onToolReady.bindStopClick": {
+                "this": "{that}.dom.sonifierStop",
+                "method": "click",
+                "args": ["{chartAuthoringInterface}.sonifier.stopSonification"]
+            }
         },
-        invokers:{
-            "updateDataEntryPanel": {
-                funcName: "floe.chartAuthoring.updateDataEntryPanel",
+        invokers: {
+            "updateDataEntryPanelFromDataSet": {
+                funcName: "floe.chartAuthoring.updateDataEntryPanelFromDataSet",
                 args: ["{that}", "{arguments}.0"]
             },
             "resetDataEntryPanel": {
-                funcName: "floe.chartAuthoring.updateDataEntryPanel",
+                funcName: "floe.chartAuthoring.updateDataEntryPanelFromDataSet",
                 args: ["{that}", []]
+            },
+            "updateActiveElements": {
+                funcName: "floe.chartAuthoring.updateActiveElements",
+                args: ["{that}"]
+            },
+            "highlightActiveLegendRow": {
+                funcName: "floe.chartAuthoring.highlightChange",
+                args: ["{that}", "{that}.chartAuthoringInterface.pieChart.legend.rows"]
+            },
+            "highlightActivePieSlice": {
+                funcName: "floe.chartAuthoring.highlightChange",
+                args: ["{that}", "{that}.chartAuthoringInterface.pieChart.pie.paths"]
+            }
+        },
+        model: {
```

Comment:
@waharnum, seems to me the model paths of ```activeRowId``` and ```activeSliceId``` are better belong to the sub-component ```pieChart``` where ```activeRowId``` will be passed down to its ```legend``` sub-component and ```activeSliceId``` will be passed down to ```pie``` sub-compnent. 

The work flow is like, the model change at the ```sonifier.model.currentlyPlayingData``` is model-relayed to the ```pieChart.model.activeRowId``` and ```pieChart.model.activeSliceId`` (what ```updateActiveElements()``` currently does). ```pie``` and ```legend``` components respond to those model changes to highlight the corresponding pie slice and legend row. Let me know if this makes sense. Thanks.